src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -384,6 +385,13 @@ public FieldDefinition setPrincipalConceptSearchExpression(String ontologyName,
         return this;
     }
 
+    public FieldDefinition setTextChoiceValues(List<String> values)
+    {
+        Assert.isTrue(getType().equals(ColumnType.TextChoice), "Type is not set to 'ColumnType.TextChoice'.");
+        setValidators(List.of(new FieldDefinition.TextChoiceValidator(values)));
+        return this;
+    }
```

Comment:
Non-standard assert.
```suggestion
    public FieldDefinition setTextChoiceValues(List<String> values)
    {
        Assert.assertEquals("Invalid field type for text choice values.", ColumnType.TextChoice, getType());
        setValidators(List.of(new FieldDefinition.TextChoiceValidator(values)));
        return this;
    }
```

Reply:
Oops! Updated.